### PR TITLE
implement first pass ETS for WNM based on ATS

### PR DESCRIPTION
**Pull request by tomkralidis**
In support of https://github.com/wmo-im/wis2-notification-message/issues/57

cc @josusky

### CODE REVIEW

File: README.md
Comment (antje-s) [inline README.md:1]: Shouldn't we use file.json for WNM?

Comment (tomkralidis) [reply README.md:1]: Good catch @antje-s !  Fixed.

File: pywis_pubsub/ets.py
Comment (antje-s) [inline pywis_pubsub/ets.py:1]: Maybe it would be good to summarize all tests and test results covered by validity as one result.

**Review by antje-s — APPROVED** [review]
ets-comment: perhaps an improvement/adjustment (not sure)

**Comment by tomkralidis** [comment]
Merging this PR; @antje-s let's discuss in another issue/feature?

**Pull request merged**